cli/replicaset/cartridge.go

```diff
@@ -80,10 +83,19 @@ func (c *CartridgeInstance) GetReplicasets() (Replicasets, error) {
 	return recalculateMasters(replicasets), nil
 }
 
+// Expel is not supported for a single instance by the Cartridge orchestrator.
+func (c *CartridgeInstance) Expel(name string) error {
+	return newErrExpelByInstanceNotSupported(OrchestratorCartridge)
+}
+
 // CartridgeApplication is an application with the Cartridge orchestrator.
 type CartridgeApplication struct {
 	runningCtx running.RunningCtx
 	preferred  connector.Evaler
+	// The cached result. There is no need to re-discovery a replicasets
```

Comment:
```suggestion
	// The cached result. There is no need to re-discover a replicasets
```

Reply:
I rewrote the cache some way, so this comment is removed now.